client/components/Gantt.jsx

```diff
@@ -0,0 +1,47 @@
+import { Chart } from "react-google-charts";
+
+export default function Gantt(props) {
+  const { data, trackHeight, width, selectCallback } = props;
+
+  return (
+    <Chart
+      height={`${data.length * trackHeight + 50}px`}
+      width={`${width * 0.8}px`}
```

Comment:
We'll have to look into how things will be sized so that the charts can be compatible with all devices, but this is good for now